Backend/src/test/java/ktsnvt/tim1/services/EventServiceUnitTests.java

```diff
@@ -0,0 +1,727 @@
+package ktsnvt.tim1.services;
+
+import ktsnvt.tim1.DTOs.*;
+import ktsnvt.tim1.exceptions.EntityAlreadyExistsException;
+import ktsnvt.tim1.exceptions.EntityNotFoundException;
+import ktsnvt.tim1.exceptions.EntityNotValidException;
+import ktsnvt.tim1.mappers.EventDayMapper;
+import ktsnvt.tim1.mappers.EventMapper;
+import ktsnvt.tim1.model.*;
+import ktsnvt.tim1.repositories.EventRepository;
+import ktsnvt.tim1.repositories.LocationRepository;
+import ktsnvt.tim1.repositories.MediaFileRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+import org.powermock.reflect.Whitebox;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Random;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
WebEnvironment.MOCK should be used for unit testing, RANDOM_PORT for integrational